docs/src/man/split_apply_combine.md

```diff
@@ -28,6 +28,12 @@ each subset of the `DataFrame`. This specification can be of the following forms
 
 All forms except 1 and 6 can be also passed as the first argument to `map`.
 
+Three are two special rules that apply to `cols => function` syntax:
+1. if `cols` is wrapped in `AsTable` object then a `NamedTuple` containing columns
+   selected by `cols` is passed to `function`
+2. if `function` is wrapped in a `ByRow` object, then it will be passed values from single
```

Comment:
Is it really worth mentioning `ByRow` for `by`? It's rarely useful as it completely ignores grouping. It's good to support it for consistency, but having it in the docstring sounds enough. Then `AsTable` can be mentioned in point 2, when the kind of arguments the function takes is mentioned.

Reply:
agreed (`ByRow` works because it exists). removed. `AsTable` is mentioned and another example is present.

Reply:
It's still there. :-)

Reply:
Ah - removed now (I have removed it in other place).